corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -176,6 +188,14 @@ hqDefine('cloudcare/js/utils', [
                         currentProgress += 1;
                     }
                 }, 250);
+            }
+        });
+    };
+
+    var showLoading = function () {
+        hqRequire(["hqwebapp/js/toggles",], function (toggles) {
```

Comment:
This cannot be a normal import?

Reply:
Good point. I think it probably can use `hqImport`.

Reply:
Yeah, got expected test failure `AssertionError: hqImport used in RequireJS modules`